FunctionalTableDataDemo/CollectionExampleController.swift

```diff
@@ -35,6 +36,14 @@ class CollectionExampleController: UICollectionViewController {
 			return LabelCell(
 				key: "id-\(index)",
 				style: CellStyle(backgroundColor: .lightGray),
+				actions: CellActions(
+					canSelectAction: { callback in
+						callback(true)
```

Comment:
It seems the value of this action is ignored in FunctionalCollectionData, using `false` will won't prevent selection